crates/ika-core/src/dwallet_mpc/crytographic_computation/mpc_computations/reconfiguration.rs

```diff
@@ -132,13 +133,19 @@ pub(crate) fn instantiate_dwallet_mpc_network_encryption_key_public_data_from_re
                     access_structure,
                 )
                 .map_err(DwalletMPCError::from)?;
-
+            let encryption_scheme_public_parameters = decryption_key_share_public_parameters
+                .encryption_scheme_public_parameters
+                .clone();
             let protocol_public_parameters = ProtocolPublicParameters::new::<
                 { secp256k1::SCALAR_LIMBS },
                 { FUNDAMENTAL_DISCRIMINANT_LIMBS },
                 { NON_FUNDAMENTAL_DISCRIMINANT_LIMBS },
                 secp256k1::GroupElement,
             >(
+                Default::default(),
+                Default::default(),
+                Default::default(),
+                Default::default(),
```

Comment:
use neutral here too, but why is this code duplicated anyways, can you explain me the two different flows?